### DISCUSSION TITLE
Are crawlers excluded from stats? Is there a way to see that traffic?

### DISCUSSION THREAD

**Discussion by craigschmidt** [plausible/analytics#2038] [Anything Else]
It seems that crawlers (Google, SEMRush, etc.) as excluded from the stats.  Is there a way to see how many pages have been index by these crawlers?

**Comment by metmarkosaric** [comment]
hi @craigschmidt! we're focused on web analytics so we exclude any bots, crawlers and other known non-human traffic by default. there's no way to see those type of stats in Plausible unfortunately.

**Reply by craigschmidt** [reply]
Totally understandable.  You'd want bot traffic excluded normally.
